### ISSUE TITLE
Add workflow to manage stale and irrelevant pull requests and issues

### ISSUE THREAD

**Issue by tenshiAMD** [all-contributors/all-contributors#621]
**Is your feature request related to a problem? Please describe.**
- Many pull requests and issues are stale and irrelevant nowadays which may lead to confusion for the users and the community.

**Describe the solution you'd like**
- I recommend installing [workflow](https://github.com/actions/stale) to manage pull requests and issues that are stale and irrelevant.

**Describe alternatives you've considered**
- [stalebot](https://github.com/probot/stale) (if we want probot based - not active anymore)

**Additional context**
- None.

**Comment by gr2m** [comment]
I'm okay with setting up the stale workflow, but I'd like us to go through open issues once and add labels to issues/PRs that are valid and that we don't want to get closed automatically. I think questions and bug reports that might no longer be relevant should be flagged as staled and automatically closed if nobody reacts

**Comment by tenshiAMD** [comment]
> I'd like us to go through open issues once and add labels to issues/PRs that are valid and that we don't want to get closed automatically.

@gr2m I also agree with this. Maybe we can ping them by leaving a message before we fully implement the workflow.

> I think questions and bug reports that might no longer be relevant should be flagged as staled and automatically closed if nobody reacts

@gr2m Yes, I think we can set up the workflow that way. I will make a PR for this soon. Thanks! 🎉

**Comment by tenshiAMD** [comment]
@gr2m FYI, I started adding `pinned` label so the workflow expected to skip those issues and PR. Thanks! 🎉

**Comment by Berkmann18** [comment]
> I'm okay with setting up the stale workflow, but I'd like us to go through open issues once and add labels to issues/PRs that are valid and that we don't want to get closed automatically. I think questions and bug reports that might no longer be relevant should be flagged as staled and automatically closed if nobody reacts

Agreed, and if someone were to comment on it (after it was closed), a new issue can be made or the issue itself can be re-opened (with the new versions in mind, if that's applicable).
